source/indexes.txt

```diff
@@ -0,0 +1,151 @@
+   /indexes/geospatial-index.txt
+   /indexes/unique-index.txt
+   /indexes/wildcard-index.txt
+   /indexes/clustered-index.txt
+
+Overview
+--------
+
+In this guide, you can learn how to use **indexes** with {+driver-short+}.
+Indexes can improve the efficiency of queries and add additional functionality
+to querying and storing documents.
+
+Without indexes, MongoDB must scan every document in a collection to find the
+documents that match each query. These collection scans are
+slow and can negatively affect the performance of your application. However, if an
+appropriate index exists for a query, MongoDB can use the index to limit the
+documents it must inspect.
+
+Operational Considerations
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To improve query performance, build indexes on fields that appear often in
+your application's queries and operations that return sorted results. Each
+index that you add consumes disk space and memory when active, so we recommend
+that you track index memory and disk usage for capacity planning. In addition,
+when a write operation updates an indexed field, MongoDB updates the related
+index.
+
+Since MongoDB supports dynamic schemas, applications can query against fields
+whose names are not known in advance or are arbitrary. MongoDB 4.2 introduced
+:manual:`wildcard indexes </core/index-wildcard/>` to help support these
+queries. Wildcard indexes are not designed to replace workload-based index
+planning.
+
+For more information about designing your data model and choosing indexes appropriate for your application, see the
+:manual:`Data Modeling and Indexes </core/data-model-operations/#indexes>` guide
+in the MongoDB Server manual.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_mflix.movies`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:ref:`<pymongo-get-started>`.
+
+Create an Index
+---------------
+
+MongoDB supports several different index types to support querying your data.
+The following pages describe the most common index types and provide sample
+code for creating each index type.
+
+- :ref:`pymongo-single-field-index`
+- :ref:`pymongo-compound-index`
+- :ref:`pymongo-multikey-index`
+- :ref:`pymongo-atlas-search-index`
+- :ref:`pymongo-text-index`
+- :ref:`pymongo-geospatial-index`
+- :ref:`pymongo-unique-index`
+- :ref:`pymongo-wildcard-index`
+- :ref:`pymongo-clustered-index`
+
+Remove an Index
+---------------
+
+You can remove any unused index except the default unique index on the
+``_id`` field.
+
+The following sections show how to remove a single index or to remove all
+indexes in a collection.
+
+Remove a Single Index
+~~~~~~~~~~~~~~~~~~~~~
+
+Pass an instance of an index or the index name to the ``drop_index()`` method to
+remove an index from a collection.
+
+The following snippet removes an ascending index on the ``title`` field
```

Comment:
Just my personal preference, but I hate the word "snippet" 😡 (also not used on most other pages in PyMongo)

Applies throughout if you do make the change